pallets/medical-record/src/tests.rs

```diff
@@ -35,7 +39,7 @@ fn patient_can_add_record() {
 
 		let records =
 			MedicalRecord::records(origin_to_account_id(o.clone()), UserType::Patient).unwrap();
```

Comment:
[Nitpicking]
Could you please change the unwrap() to a expect() just for the sake of clarity?